src/screens/shared/farmProfile/FarmProfile.js

```diff
@@ -23,11 +27,13 @@ class FarmProfile extends React.Component {
     const { match } = this.props;
     const { farmId } = match.params;
     const farm = await getSingleFarm(farmId);
-    this.setState({ farm, farmId, loading: false });
+    const GAPContact = await getUserById(farm.groupGapContactId);
```

Comment:
handle case where there is no group gap contact (page is currently erroring when farm.groupGapContactId is null)

phoebe is getting the details from spring about how to display this information

Reply:
no icon when group contact is null